src/extensions/isolator/isolator.ts

```diff
@@ -86,13 +86,15 @@ export default class Isolator {
     await writeComponentsToCapsules(components, graph, capsules, capsuleList, this.packageManager.name);
 
     const after = await getPackageJSONInCapsules(capsules, packageManager);
-    const toInstall = capsules.filter((item, i) => {
-      return (
-        !equals(before[i], after[i]) ||
-        after[i].packageManager === '' ||
-        !isOldPackageManager(after[i].packageManager, config, packageManager)
-      );
-    });
+    const toInstall = capsules
+      .filter((item, i) => {
+        return (
+          !equals(before[i], after[i]) ||
+          after[i].packageManager === '' ||
+          !isOldPackageManager(after[i].packageManager, config, packageManager)
```

Comment:
@qballer , let's change this function name on your branch. Maybe isPackageManagerIsTheSame